### PR TITLE
Fix `collections.abc.MappingView` subclass persistence.

### PR DESCRIPTION
**Pull request by agroszer**

### CODE REVIEW

File: src/pjpersist/serialize.py [hunk 1]
```diff
@@ -342,6 +355,11 @@ def get_state(self, obj, pobj=None):
             return {'_py_type': 'datetime.datetime',
                     'value': obj.strftime(FMT_DATETIME)}
 
+        # dunno... check type by type or stick with subclasses of MappingView?
```
Comment (strichter) [inline src/pjpersist/serialize.py:358]: This is not a good comment to leave in the code. State confidently the case you are covering.

File: src/pjpersist/tests/test_serialize.py [hunk 2]
```diff
@@ -290,7 +306,7 @@ def doctest_ObjectWriter_get_non_persistent_state():
 
     A simple old-style class:
 
-      >>> class That(object):
+      >>> class That():
```
Comment (strichter) [inline src/pjpersist/tests/test_serialize.py:309]: `()` not needed.

**Review by strichter — APPROVED** [review]
Code is good, just the comment needs to change.

**Comment by agroszer** [comment]
we decided to convert the *View objects to a list on write

**Pull request merged**